task/secure/src/main.rs

```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#![feature(naked_functions)]
+#![no_std]
+#![no_main]
+
+use hypocalls::*;
+#[allow(unused_imports)]
+use userlib::*;
+
+#[link_section = ".tz_table"]
+#[no_mangle]
+#[used]
+static TZ_TABLE: SecureTable = SecureTable {
+    magic: 0xaaaa_bbbb,
+    write_to_flash: write_to_flash,
+};
+
+#[export_name = "main"]
+fn main() -> ! {
+    loop {
+        cortex_m::asm::nop();
```

Comment:
What is going on here? This will burn all cycles at its priority level and starve anything below.

Reply:
Added details in README (tl;dr this isn't really a "task")